### DISCUSSION TITLE
Show if a parking has fee or is free

### DISCUSSION THREAD

**Discussion by pluja** [organicmaps/organicmaps#1064] [Ideas]
OpenStreetMaps has a field for this information, and I would love to be able to see if a Parking is free or a fee needs to be paid. Also, being able to filter parkings by either they are free of paid parkings.

**Comment by biodranik** [comment]
That's not easy for many cases. For example, in some countries, the parking may be free for local residents, and free for visitors for up to one or two hours. Good analysis and UI design are needed to make this feature really usable.

**Reply by bentolor** [reply]
https://github.com/organicmaps/organicmaps/discussions/1351 might be related as I undestood this idea is centered around the OSM Tag [`fee=no`]( https://taginfo.openstreetmap.org/keys/fee)

**Comment by vbertola** [comment]
This would be a useful feature as long as users understand that in many countries this kind of information is incomplete or imprecise, so they'd always have to check. You have at least five categories - access restricted by category (e.g. access=permit, access=private), access restricted by usage (e.g. access=customers, access=destination), public access with fee (no access tag, fee=yes), public access without fee (no access tag, fee=no), and unknown (no access tag and no fee tag, or unclear/complex values). You could find ways of showing these (e.g. a small $ sign for fee, and shaded variants for access) but you would need to make sure that they are easily understood.

**Comment by patepelo** [comment]
This feature is now partly covered. Please tell us what you think.

**Reply by vbertola** [reply]
I saw "P$" signs appear on the maps, that's great, thanks :) Of course it suffers from the fact that not so many mappers bother to add fee=yes or fee=no, but I think it's already useful to see it where the information is present, and the more it appears the more mappers will take care of providing it.

**Reply by biodranik** [reply]
@patepelo please investigate and create an issue if necessary on how to easily edit the "free/paid" status of objects in OM.

**Reply by patepelo** [reply]
New issue opened accordingly: https://github.com/organicmaps/organicmaps/issues/6655#issue-2008147602
